### DISCUSSION TITLE
Current standard scoring is still extremely combo heavy

### DISCUSSION THREAD

**Discussion by GopoBoi** [ppy/osu#18260] [Ideas]
Mainly due to how the accuracy part is handled. Since acc is linear, 99.9% of plays on leaderboards will get around 90% - 95% or more of the accuracy component every time, since plays below 90% acc are almost exclusive to players playing maps above their skill level, so they don't get enough combo to matter. Which means that even getting 100% accuracy vs a player with 90% gives you just 30k score more, which the other player can get by just getting 5% more combo.
Even if standard scoring was to be adjusted to a 50/50 split or further, it would only be a very small improvement, since 90% of the accuracy portion is basically free score.
 
A solution that I can think of is either a very accuracy heavy ratio - something like 70 acc /30 combo (this would make it so around 23% of combo would be needed to overcome a 10% acc difference), but this would also mean that the majority of scores would never go below ~700k points.
Or having the accuracy portion be exponential, which as I understand isn't really wanted, as it would make score more difficult to understand.

**Comment by smoogipoo** [comment]
Probably need to find a ratio that yields similar results to osu-stable's ScoreV2. Combo's worth more in lazer due to the way it's factored in.

**Comment by WitherFlower** [comment]
I don't really understand why having exponential/polynomial scaling for accuracy isn't wanted. I'm pretty sure everyone can understand that the higher your accuracy is, the harder it gets to gain accuracy (ie. it's easier to go from 60% to 61% than from 99% to 100%).

I think such a scaling, combined with a 50/50 combo/acc split could solve this issue very easily.

I'd also like to mention that scoreV2 already uses a similar scaling (acc^10).

**Reply by bdach** [reply]
Scoring is meant to be simple. Proportional scoring is easier to understand than polynomial/exponential. Classic is still available as an option for people who prefer the latter.

**Reply by smoogipoo** [reply]
That being said, if we're going to explain the system in simple terms then there's not much of a difference between:
`acc_score = acc_portion * acc%`
and
`acc_score = acc_portion * (acc%)^10`

Still not sure how I feel about the linear scaling myself. I think the primary reason for why exponential scaling isn't used last time was one of impact (at low acc%).

**Reply by Kaerulans** [reply]
> Scoring is meant to be simple. Proportional scoring is easier to understand than polynomial/exponential. Classic is still available as an option for people who prefer the latter.

Classic scoring does nothing to address this issue. It just increases the score cap to be similar to stable instead of being capped at 1mil, but the acc portion is still 30% of the score and it still scales linearly.

**Comment by Kaerulans** [comment]
This is honestly my biggest issue with lazer at the moment. I think everyone can agree that getting a 99% 1 miss play where the miss is 95% into the map should be worth more score than a 90% fc. Almost all notable plays have acc above 90%, therefore the differences in most notable plays due to acc is just 30k score. The solution in score v2 (acc% ^10) works really well in my opinion.
